spec/services/update_duplicate_matches_spec.rb

```diff
@@ -22,6 +22,22 @@
   end
 
   describe '#save!' do
+    context 'when existing fraud match' do
```

Comment:
Maybe expand this `context` to reflect the contents of the `do` block: the candidates have been manually unblocked

Reply:
Done!